app/frontend/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
+    "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js --host 0.0.0.0",
```

Comment:
I would like to try getting this to work without this (I know this was my change but I don't think it's a good idea).

Reply:
@Sybrand Mind stepping in here?

Reply:
another script like "npm run container" could use --host 0.0.0.0 so that "npm run dev" continues to bind to localhost only